sdk/attestation/azure-security-attestation/azure/security/attestation/_client.py

```diff
@@ -133,50 +172,112 @@ def attest_sgx_enclave(self, quote, inittime_data=None, runtime_data=None, **kwa
+            ),
+            token,
+        )
 
     @distributed_trace
-    def attest_open_enclave(self, report, inittime_data=None, runtime_data=None, **kwargs):
-        # type:(bytes, AttestationData, AttestationData, Dict[str, Any]) -> AttestationResponse[AttestationResult]
-        """ Attests the validity of an Open Enclave report.
-
-        :param bytes report: An open_enclave report generated from an Intel(tm) SGX enclave
-        :param inittime_data: Data presented at the time that the SGX enclave was initialized.
-        :type inittime_data: azure.security.attestation.AttestationData 
-        :param runtime_data: Data presented at the time that the open_enclave report was created.
-        :type runtime_data: azure.security.attestation.AttestationData 
-        :keyword draft_policy: "draft" or "experimental" policy to be used with
-            this attestation request. If this parameter is provided, then this 
+    def attest_open_enclave(
+        self, report, **kwargs  # type: Dict[str, Any]
+    ):  # type: (...) -> Tuple[AttestationResult, AttestationToken]
+        """Attests the validity of an Open Enclave report.
+
+        :param bytes report: An open_enclave report generated from an Intel(tm)
+            SGX enclave
+        :keyword bytes inittime_data: Data presented at the time that the SGX
+            enclave was initialized.
+        :keyword bytes inittime_json: Data presented at the time that the SGX
+            enclave was initialized, JSON encoded.
+        :keyword bytes runtime_data: Data presented at the time that the open_enclave
+            report was created.
+        :keyword bytes runtime_json: Data presented at the time that the open_enclave
+            report was created. JSON Encoded.
+        :keyword str draft_policy: "draft" or "experimental" policy to be used with
+            this attestation request. If this parameter is provided, then this
             policy document will be used for the attestation request.
             This allows a caller to test various policy documents against actual data
             before applying the policy document via the set_policy API.
 
-        :paramtype draft_policy: str
+        :keyword bool validate_token: if True, validate the token, otherwise return the token unvalidated.
+        :keyword validation_callback: Function callback to allow clients to perform custom validation of the token.
+            if the token is invalid, the `validation_callback` function should throw
+            an exception.
+        :paramtype validation_callback: ~typing.Callable[[AttestationToken, AttestationSigner], None]
+        :keyword bool validate_signature: if True, validate the signature of the token being validated.
+        :keyword bool validate_expiration: If True, validate the expiration time of the token being validated.
+        :keyword str issuer: Expected issuer, used if validate_issuer is true.
```

Comment:
nit: for consistency in adding \`s around parameter references (which I think is a nice little garnish in docstrings). This applies to all instances of `issuer` documentation, and possibly other parameter references
```suggestion
        :keyword str issuer: Expected issuer, used if `validate_issuer` is true.
```